admin.py

```diff
@@ -0,0 +1,183 @@
+def choose_the_option():
+    option = int(input("Вибери спеціальність: \n" "1. Адмін\n"))
+    return option
+
+
+option = choose_the_option()
+
+drinks = {
+    "Лагер": 30,
+    "Ель": 35,
+    "Оболонь": 28,
+    "Чернігівське": 30,
+    "Львівське": 32,
+    "Обрій": 31,
+    "Горілка-Хортиця": 180,
+    "Горілка-Хлібна слобода": 200,
+    "Горілка-Вознесенська": 150,
+}
+
+
+def function_of_admin():
+    if option == 1:
+        choose_Admin = int(
+            input(
+                "Опції Адміна:\n"
+                "1. Додати до асортименту напій\n"
+                "2. Видалити з асортименту напій\n"
+                "3. Змінити ціну на товар\n"
+                "4. Прийняти на роботу працівника\n"
+                "5. Звільнити з роботи працівника\n"
+                "6. Змінити заробітню плату працівнику\n"
+                "7. Переглянути список робітників\n"
+                "8. Переглянути список напоїв\n"
+                "9. Переглянути замовлений товар\n"
+                "10. Замовити напій\n"
+                "11. Назад\n"
+                "12. Вийти\n"
+            )
+        )
+        return choose_Admin
+
+
+choose_Admin = function_of_admin()
+
+
+def add_or_remove_drink():
```

Comment:
It is necessary to separate the functionality. There is a function to add a drink. And there is a function to remove the drink.